src/main/java/com/uid2/admin/vertx/service/ClientKeyService.java

```diff
@@ -437,4 +447,17 @@ private void handleClientRename(RoutingContext rc) {
             rc.fail(500, e);
         }
     }
+
+    private int getServiceId(RoutingContext rc) {
+        Integer serviceId = 0;
```

Comment:
This can be `int` since we don't want null `serviceId` anyway

Reply:
Done